pkg/providers/snow/clientregistry.go

```diff
@@ -0,0 +1,38 @@
+package snow
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aws/eks-anywhere/pkg/aws"
+)
+
+type ClientRegistry interface {
```

Comment:
nit: I would define this next to where it's used

Reply:
I have it defined here because it is used in both validator and defaulter